nntrainer/app_context.cpp

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file	app_context.cpp
+ * @date	10 November 2020
+ * @brief	This file contains app context related functions and classes that
+ * manages the global configuration of the current environment
+ * @see		https://github.com/nnstreamer/nntrainer
+ * @author	Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug		No known bugs except for NYI items
+ *
+ */
+#include <dirent.h>
+#include <iostream>
+#include <sstream>
+
+#include <app_context.h>
+#include <nntrainer_log.h>
+#include <util_func.h>
+
+namespace nntrainer {
+
+/**
+ * @brief initiate global context
+ *
+ */
+static void init_global_context_nntrainer(void) __attribute__((constructor));
+
+/**
+ * @brief finialize global context
+ *
+ */
+static void fini_global_context_nntrainer(void) __attribute__((destructor));
+
+static void init_global_context_nntrainer(void) {}
```

Comment:
may this function should configure the global appcontext working path as cwd?

Reply:
`getWorkingPath` will return relative path if is not set explicitly to give user flexibility. (e.g call `chdir()` later on user's code)